erdgo/cmd/testTransaction/main.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-sdk/erdgo"
+)
+
+func main() {
+	ep := erdgo.NewElrondProxy("http://174.138.103.62:8079")
```

Comment:
do we need to expose this specific ip?
applies to other files as well

can't we use a proxy on localhost for test?

Reply:
Done